app/extensions/administration/models.py

```diff
@@ -0,0 +1,55 @@
+"""Bot database models."""
+import typing
+
+import motor.motor_asyncio
+
+db = motor.motor_asyncio.AsyncIOMotorClient('mongo').crabot
+
+
+class Server:
+    """Database guild(server) model."""
+
+    @staticmethod
+    async def get_or_create(id_: typing.Any):
+        """Retrieve (or create and retrieve) guild from the database."""
+        created = False
+        server = await db.servers.find_one({'id': id_})
+        if not server:
+            server = {
```

Comment:
There is no reason for the record to exist until the autorole is added. Moreso, I would work off the `autoroles` collection instead, not the servers/guilds.